package.json

```diff
@@ -15,6 +15,7 @@
     "@types/react": "18.2.21",
     "@types/react-dom": "18.2.7",
     "@typescript-eslint/eslint-plugin": "^6.7.0",
+    "@vercel/postgres": "^0.5.1",
```

Comment:
Because we're using Nextjs we get access to vercel which is currently hosting our app. Out of the box we're given the option to use a postgres database which vercel sets up and we can configure.

Our app is "serverless" because we're relying on vercel to handle that heavy lifting. This also means our api endpoints look different. More on that in the next comments